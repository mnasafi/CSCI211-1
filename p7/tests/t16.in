echo t16: Insert days of week into tree, then call distance
insert Friday
insert Monday
insert Saturday
insert Sunday
insert Wednesday
insert Thursday
insert Tuesday
distance
