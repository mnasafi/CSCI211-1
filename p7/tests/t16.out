t16: Insert days of week into tree, then call distance
Average distance of nodes to root = 3
